src/components/Task.vue

```diff
@@ -221,6 +249,16 @@ hr {
   height: 21px;
 }
 
+.avatar {
```

Comment:
I'm guessing copied from Asana? If so, let's add a comment (and this will move to the new component)